src/services/ContributorAuditArgumentParser.ts

```diff
@@ -8,7 +8,7 @@ import BitbucketCloudContributorAuditProvider from "./ContributorAuditService/pr
 interface IContributorAuditArguments extends ICommonArguments {
   days: number;
   secret: string;
-  saveResults: boolean;
+  saveResults: ScmResultsFormat;
```

Comment:
Maybe we should rename this param to something else

Reply:
`--saveResults json` is ok - maybe support `--saveResults` which defaults to txt as well